alphalens/performance.py

```diff
@@ -247,7 +247,11 @@ def quantize_factor(factor, quantiles=5, by_group=False):
     """
 
     def quantile_calc(x, quantiles):
-        return pd.qcut(x, quantiles, labels=False) + 1
+        try:
+            return pd.qcut(x, quantiles, labels=False) + 1
+        except ValueError:
+            # This is supposed to fix "ValueError: Bin edges must be unique"
+            return pd.qcut(x.rank(method='first'), quantiles, labels=False) + 1
```

Comment:
I suppose the error occurs when there are not enough unique values to generate the quantiles. `.rank()` then probably makes them unique? Not sure I follow that. Also, if that's the case I think a better approach would be raise an exception that there are not unique values to generate the requested number of quantiles. Or at least a warning that we're doing some changes.